examples/kft-v2/trl-gpt2-checkpointing.ipynb

```
@@ -0,0 +1,1215 @@
+    "    func=trl_train,\n",
+    "    num_nodes=2,  # Distributed training across 2 nodes\n",
+    "    resources_per_node={\n",
+    "        \"cpu\": \"2\",\n",
+    "        \"memory\": \"4Gi\",\n",
+    "        # Uncomment for GPU training:\n",
+    "        # \"nvidia.com/gpu\": \"1\",\n",
+    "    },\n",
+    "    packages_to_install=[\n",
+    "        \"transformers[torch]\",\n",
+    "        \"trl\", \n",
+    "        \"peft\", \n",
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)